tests/python/release_checklist/check_graph_view.py

```diff
@@ -15,18 +15,12 @@
 * `graph` has directed edges, while `graph2` has undirected edges.
 * `graph` and `graph2` are shown in two different viewers.
 * There is a third viewer, `Both`, that shows both `graph` and `graph2` in the same viewer.
-* The `coincident` viewer shows two nodes, `A` and `B`, at the same position
```

Comment:
less manual testing 🥳

Reply:
Yeah! For some reason, I still have some minor non-determinism in `fjadra` so I can't really get rid of the remaining checks, but I'm looking into that separately.